frontend/app/src/views/source-data/ProcessControlView.vue

```diff
@@ -215,7 +227,10 @@ export default {
       const tableHeaders = this.formFieldHeaders.map((h) => {
```

Comment:
I guess filtering can be done directly in map function as below. See if this works 
const tableHeaders = this.formFieldHeaders
    .filter(header => header !== 'controlTableId') // Filter out 'controlTableId'
    .map((h) => {
        return { title: h, key: h };
    });

Reply:
In ideal situation YES it should be like that, but this.formFieldHeaders is State variable and returning from the Vue await this.fetchFormFieldHeaders(); and took some time for a reason to return expected values. When I combine Map and Filter it just drop the actions column as it is locally defined there.
Thats why I filter out on both the variable (ARRAY) in next step:
this.headers = [...tableHeaders, ...this.headers].filter(...)